docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+  chart-module:
+    image: seungprk/chart-module:latest
+    depends_on:
```

Comment:
I'm wondering if you were to change your chart module to depend on seeder if it will run in this order
mongodb -> seed db -> chart-module 
whether it will alleviate that issue you have to have your db attempt multiple connections to seed.

Reply:
That might be worth trying, but I have a suspicion that will not alleviate the race condition issue, as depends_on does not wait for the database to be ready to execute the next container.
https://docs.docker.com/compose/compose-file/#depends_on

Reply:
I see. An interesting problem to solve. Re-attempting db connection seems to be the recommended practice in production.